src/manifest.json

```diff
@@ -24,6 +25,11 @@
         "default_title": "PoE Wiki Search",
         "default_popup": "ui/ui.html"
     },
+    "browser_specific_settings": {
+        "gecko": {
+          "id": "addon@example.com"
```

Comment:
Needed to make use of storage, see
https://extensionworkshop.com/documentation/develop/extensions-and-the-add-on-id/#when-do-you-need-an-add-on-id

**TODO:** we need to decide on a real ID